CHANGELOG.md

```diff
@@ -5,9 +5,17 @@ The format is based now on [Keep a
 Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to
 [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [3.0.4]
```

Comment:
It seems we must change at least a minor version, because we change env param (UPLOADCARE_PRIVATE_KEY -> UPLOADCARE_SECRET_KEY)
And, please, add info about it here

Reply:
Environment name parameter can get any name — it's just an example in readme file. All will be fine with any names — only order matters.

But we **should** change the minor version because of break backwards  compatibility — this version will not work in same environment as previous in case of php < 7.1.